main.js

```diff
@@ -95,16 +94,16 @@ function clickStar() {
   }
 }
 
-function redStar(parentCard){
-  var whiteStar = parentCard.childNodes[1].childNodes[1];
-  var redStar = parentCard.childNodes[1].childNodes[3];
-  whiteStar.classList.add('hidden');
-  redStar.classList.remove('hidden');
+function orangeStar(parentCard){
+  var starImage = parentCard.childNodes[1].childNodes[1];
+  starImage.src = "./assets/star-active.svg"
 }
 
 function whiteStar(parentCard){
-  var whiteStar = parentCard.childNodes[1].childNodes[1];
```

Comment:
I wonder if we can use `.first()` to find the closest `img` child element. I researched some and it appears to go the opposite direction of `.closest()`.